cmd/lotus-miner/actor_test.go

```diff
@@ -34,8 +34,8 @@ func TestWorkerKeyChange(t *testing.T) {
 
 	kit.QuietMiningLogs()
 
-	blocktime := 1 * time.Millisecond
-	client1, client2, miner, ens := kit.EnsembleTwoOne(t, kit.MockProofs())
+	blocktime := 5 * time.Millisecond
+	client1, miner, ens := kit.EnsembleMinimal(t, kit.MockProofs())
```

Comment:
There was zero reason this test had two nodes, and with the really low block time, on lower-perf instance it was possible that the second node never caught up